wheretofindme/views.py

```diff
@@ -25,6 +26,8 @@ class UserProfileView(DetailView):
     def get_context_data(self, *args, **kwargs):
         context = super().get_context_data(*args, **kwargs)
         context["identities"] = context["object"].internetidentity_set.all()
+        context["here"] = self.request.build_absolute_uri()
```

Comment:
In #22, I hardcoded `url(/static/images/...)` into a stylesheet. Should I be doing this, instead, in that PR?

Reply:
Oh, so you did. I think in a stylesheet you have the static files in known positions relative to each other, so `url(../images/foo)` should work, no? In any case, unless they're in known relative positions, I'd prefer to use the `static` templatetag (either in a view like this, or directly in a template) because it can handle swapping out our static files storage gracefully.

Reply:
Relative URLs should work, yeah. I'll make that change.